content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -169,6 +169,7 @@ different Kubernetes components.
 | `StorageVersionHash` | `true` | Beta | 1.15 | |
 | `Sysctls` | `true` | Beta | 1.11 | |
 | `TTLAfterFinished` | `false` | Alpha | 1.12 | |
+| `IndexedJob` | `false` | Alpha | 1.21 | |
```

Comment:
I think these are alphabetically ordered

Reply:
ah, missed that. Fixed.